samples/core/console-apps/Fibonacci/README.md

```diff
@@ -1,7 +1,7 @@
 Fibonacci Sample
 ================
 
-This sample is part of the [step-by-step tutorial](https://docs.microsoft.com/dotnet/articles/core/tutorials/using-with-xplat-cli)
+This sample is part of the [step-by-step tutorial](https://docs.microsoft.com/dotnet/core/tutorials/using-with-xplat-cli)
```

Comment:
@mairaw I need to get my eyes checked. I thought they were in the main doc set.  Not sure what I was seeing...